### DISCUSSION TITLE
[PSD6] Example 3

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/psd2223#143] [Esercizi (A-L)]
Semplificare la seguente equazione booleana:

![image](https://user-images.githubusercontent.com/69389461/210016364-43927d37-08a1-4818-9d7f-b37425801d99.png)

**Comment by Elia-Belli** [comment]
$Y = \bar A B C + \bar A = \bar A$ (assorbimento)

**Comment by FeddyLix17** [comment]
corretto
